README.md

```diff
@@ -74,11 +74,22 @@ git@github.com:ita-social-projects/StreetCode.git
   ```
 
 ### Setup
-  1. Change connection string  
-   (Go to **appsettings.json** and write your local database connection string)
-  2. Create local database  
-   (Run project and make sure that database was created filled with data)
-
+  **1. Change connection string**  
+   * Go to `appsettings.Local.json` in **StreetCode.WebApi** project and write your local database connection string in following format:
+    
+     ```
+      Server={local_server_name};Database=StreetcodeDb;User Id={username};Password={password};MultipleActiveResultSets=true;
```

Comment:
please add TrustServerSertificate=True (google it for more specific information)